recipes/python-lsp-server/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "python-lsp-server" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8f4b8a44300839b22e6974500d5e7627ffa79c841c2ed6c55918f7b43eb375cf
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - pylsp = pylsp.__main__:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - jedi >=0.17.2,<0.18.0a0
+    - python-lsp-jsonrpc >=1.0.0
+    - pycodestyle >=2.6.0,<2.7.0
```

```suggestion
    - pycodestyle >=2.7.0
```

This is because the latest version of `autopep8` requires this version. To do this change, I think we you need to release 1.0.1 of `python-lsp-server`.